app/Http/Controllers/Admin/AdminCheckoutController.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Checkout;
+use App\Http\Controllers\Controller;
+use App\PaymentType;
+use App\Semester;
+use App\Transaction;
+use App\User;
+use Carbon\Carbon;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Validator;
+
+class AdminCheckoutController extends Controller
```

Comment:
There are a lot of code duplications. I am okay if it goes in now, but please open an issue to create a Checkout util class (like we have for printer and tabulator).